src/less/box-shadow.less

```diff
@@ -4,30 +4,42 @@
 // Opinionated defaults for box shadows
 
 @boxShadowDefaultXOffset: 0;
-@boxShadowDefaultYOffset: @size_4xs;
```

Comment:
Removed, checked that there were no usages of this in the Clever codebase by searching with Github: `org:Clever boxShadowDefaultYOffset`